src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -211,7 +211,7 @@ $item-height: 32px;
   position: relative;
   display: flex;
   align-items: center;
-  border-radius: $border-radius;
+  padding-left: calc(var(--card-horizontal-spacing) / 2);
```

Comment:
I feel like the `--card-horizontal-spacing` var no longer serves the purpose it was meant for. We should do some cleanup.

I also dont understand why we use half of the spacing here. It feels like a happy coincidence.

Reply:
I tried using better and self-explanatory vars in this new commit: https://github.com/apple/swift-docc-render/pull/91/commits/5d63fd14d364ba4caa4911d01a2d5cca848a136a